docs/network_architecture.rst

```diff
@@ -53,6 +53,74 @@ Federated Machine Learning Flow
     b. Resume the task
 7. Once all rounds are completed the *Coordinator* can just exit
 
+Model weights distribution
+--------------------------
+
+Models can be massive (several dozens of megabytes), and protobuf is
+not suited for exchanging such data. Instead, the participants and the
+coordinator use an S3 bucket to exchange their weights. The exact
+mechanism is represented by the sequence diagram below.
+
+At the beginning of a round (1) the selected participants send a
+``StartTrainingRound`` request.
```

Comment:
does the double back tick work in rst files for this and other variables? otherwise we might have to type it like
```
:code:`StartTrainingRound`
```

Reply:
It seems to work yeah